client/components/ProductOverview/ProductOverview.jsx

```diff
@@ -6,7 +6,7 @@ import ImageGallery from './ImageGallery';
 
 import '../../css/ProductOverview.scss';
 
-const ProductOverview = ({ productId }) => {
+const ProductOverview = ({ currentProduct, productId }) => {
```

Comment:
I passed in currentProduct just to see if it was updating when user clicks on related product card. Feel free to use/modify what you need for the remaining sections of Product Overview.

Reply:
I am currently making an axios call with the productId to get the main product. But I can now refactor to just use the currentProduct passed in, thanks.